metaflow/plugins/kfp/kfp_metaflow_step.py

```diff
@@ -57,12 +57,12 @@ def _step_cli(
     input_paths = None
 
     tags_extended: List[str] = [
-        f"--tag argo_workflow:{workflow_name}",
-        "--tag pod_name:$MF_POD_NAME",
-        "--tag pod_namespace:$MF_POD_NAMESPACE",
+        f"--sys-tag argo_workflow:{workflow_name}",
+        "--sys-tag pod_name:$MF_POD_NAME",
+        "--sys-tag pod_namespace:$MF_POD_NAMESPACE",
         # TODO(talebz): A Metaflow plugin framework to customize tags, labels, etc.
-        "--tag zodiac_service:$ZODIAC_SERVICE",
-        "--tag zodiac_team:$ZODIAC_TEAM",
+        "--sys-tag zodiac_service:$ZODIAC_SERVICE",
+        "--sys-tag zodiac_team:$ZODIAC_TEAM",
```

Comment:
I think it's better to move these to https://github.com/zillow/metaflow/blob/886b44daa197cb805eaa40969e6d2c0e1002c526/metaflow/plugins/kfp/kfp_decorator.py#L132-L136 instead of adding a `--sys-tag` parameter.